source/style-guide/style/voice-and-tone.txt

```diff
@@ -71,7 +72,7 @@ Best Practices
 Consider the following best practices for voice and tone:
 
 .. list-table::
-   :widths: 33 33 34
+   :widths: 50 25 25
    :header-rows: 1
 
    * - Best practices
```

Comment:
**question:** Why _Use_ and _Don't Use_ rather than _Use_ and _Avoid_ like on other pages? Is this a matter of strength of our intent / message around these issues?

Reply:
**question:** Should we link to the other pages (such as second person and imperative mood -> /style-guide/writing/write-to-the-user.txt?

Reply:
1. Hadn't thought about it, that language was already here. Changed to "avoid".
2. Added links.